### ISSUE TITLE
script-not-executable

### ISSUE THREAD

**Issue by nschloe** [TriBITSPub/TriBITS#158]
Warning from Debian:
```
script-not-executable
usr/lib/x86_64-linux-gnu/cmake/tribits/ctest_driver/makeCMakeCacheFile.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/build_docs.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/build_ref/create-build-ref.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/developers_guide/generate-dev-guide.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/developers_guide/scripts/dummy_test_commit.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/publish_docs.sh
```
See [here](https://lintian.debian.org/tags/script-not-executable.html):
```
This file starts with the #! sequence that marks interpreted scripts, but it is not executable.
Severity: normal, Certainty: certain
Check: scripts, Type: binary
```

**Comment by bartlettroscoe** [comment]
@nschloe, currently TriBITS CMake code is not copying these files, Trilinos CMake code is.  As I mentioned [here](https://github.com/TriBITSPub/TriBITS/issues/125#issuecomment-223356253), we need to work #114 which will fix this once Trilinos takes advantage of this.

Therefore, I am closing as a duplicate of #125 which will be addressed as part of #114.  The story #114 is not a huge amount of work but there are just a lot of other priorities that I have to work first (i.e. lots of other people screaming at me :-)).

If you don't mind taking a crack at #114 on a topic branch, then I could devote some time to help finish that up and get Trilinos updated.   See [Contributing to TriBITS](https://github.com/TriBITSPub/TriBITS/wiki/Contributing-to-TriBITS).

**Comment by nschloe** [comment]
Whoops, sorry for filing it twice. I'd forgotten about #125.
